### ISSUE TITLE
Can't pass arguments to node script (--harmony-proxies)

### ISSUE THREAD

**Issue by SargTeX** [Unitech/pm2#2002]
I can't pass cli arguments to the executed node script.

PM2 version: 1.0.1
Node.JS version: 4.3.2
OS: Ubuntu 14.04 LTS 64-bit
`head -n 50 ~/.pm2/pm2.log` output didn't contain any line about the test case (all entries where at least a day old)

As far as I understood, the following command should pass the cli argument "--harmony-proxies" to the "test.js" script:
`pm2 start test.js --node-args="--harmony-proxies"`

But PM2 always receives an error and immediately tries to restart the server:

    Error: Native proxies not enabled. use 'node --harmony' or 'node --harmony-proxies'
           at Object.<anonymous> (/var/www/myproject/node_modules/harmony-proxy/index.js:7:8)

The test.js file:

    var proxy = require('harmony-proxy');

The exact commands I did:
* `npm install harmony-proxy`
* `pm2 start test.js --node-args="--harmony-proxies"`

When firing `node --harmony-proxies test.js`, the script runs without an error.

**Comment by Unitech** [comment]
Can you run `pm2 update` and try again?

**Comment by SargTeX** [comment]
I already tried that, nothing changes.
I also updated PM2 via `npm install pm2@latest -g` to version `1.0.2`, and fired the start command as written above again, but I still receive the same errors.

**Comment by Unitech** [comment]
I tried locally, it works well so I guess the application is started with the wrong node.js version,

Could you do a `pm2 show test` and show me the result of the line `node.js version`

**Comment by SargTeX** [comment]
The node.js version seems to be correct: `4.3.2`

But we also see that the "v8 arguments" line is empty, so it does not receive any arguments...

I appended the complete output as a text file: [pm2 show test.txt](https://github.com/Unitech/pm2/files/162943/pm2.show.test.txt)

-- but I just found a solution for my case:

- `pm2 delete test`
- `pm2 start test.js --node-args="--harmony-proxies"`

It turned out that completely removing the test-process and restarting it with the node-args again solved it. I thought I've done that multiply times before, but maybe I put the arguments wrong.

There might be another cause for this behaviour: Doesn't PM2 restart the script as supplied, if I fire `pm2 start` on the same node script again?

**Comment by Unitech** [comment]
Ahah alright

Look like there is a small issue with configuration merging (environment merging is implemented btw), let me check that

**Comment by Unitech** [comment]
Fixed, if you want to try:

```
$ npm install Unitech/pm2#development -g
$ pm2 update
$ pm2 restart app --node-args="--new-options"
```

Landing in 1.0.3

**Comment by Unitech** [comment]
Update available (PM2@1.1.1 - latest stable):

```
$ npm install pm2@latest -g
$ pm2 update
```

Please re-open this issue if you notice any problem,
